### DISCUSSION TITLE
Add CiviWiki to list of 'digital tools for activism'

### DISCUSSION THREAD

**Discussion by brylie** [CiviWiki/OpenCiviWiki#1414] [Ideas]
The [Digital Tools for Activism](https://github.com/drewrwilson/toolsforactivism) list contains several projects that are similar in spirit to CiviWiki.

# Goals
- Increase visibility of CiviWiki project
- Connect with likeminded people/projects

# Task
- [x] Open a pull request with 'Digital Tools for Activism' that includes a link to civiwiki.org in the 'software as a service category'

**Comment by AbhijithGanesh** [comment]
@silvarthur is this issue relevant? Can you check it up and reach out to brylie

**Comment by silvarthur** [comment]
Yes, it is. Thanks @AbhijithGanesh . @brylie , could you assign me to this issue?

**Comment by silvarthur** [comment]
@brylie?

**Comment by brylie** [comment]
Assigned. Thanks!

**Comment by silvarthur** [comment]
Thank you! (:

**Comment by silvarthur** [comment]
I just opened the pull request at Digital Tools for Activism. Waiting it to be accepted.

https://github.com/drewrwilson/toolsforactivism/pull/61

**Comment by silvarthur** [comment]
How? The pull request was not accepted yet and I could not found CiviWiki opening the URL https://github.com/drewrwilson/toolsforactivism#open-source-host-it-yourself. Are you sure?

**Comment by brylie** [comment]
Sorry, I saw CiviCRM there, not CiviWiki

**Comment by silvarthur** [comment]
No problem. I will let you know as soon as the pull request is accepted.

**Comment by silvarthur** [comment]
Hello, @brylie. I am still waiting for the pull request to be accepted. It seems the guy who does that is not keeping the repository anymore. ):

**Comment by silvarthur** [comment]
Hey @brylie , the pull request was just accepted! 
https://github.com/drewrwilson/toolsforactivism/pull/61

**Reply by brylie** [reply]
Thanks @silvarthur :-)

**Comment by brylie** [comment]
@all-contributors please add @silvarthur for community
